hydromt_sfincs/sfincs.py

```diff
@@ -1342,6 +1342,9 @@ def setup_cn_infiltration_with_ks(
 
         # Done
         self.logger.info(f"Done with determination of values (in blocks).")
+        
+        # Convert ks - (e.g. from micrometer per second to mm/hr which is SFINCS required)
+        da_ks = da_ks * factor_ksat
```

Comment:
Looks fine to me, though this is also what the data_catalog is meant for (changing variable names to the expected ones, changing units etc), for example here for era5 data with unit_mult:

```
era5_hourly:
  crs: 4326
  data_type: RasterDataset
  driver: netcdf
  kwargs:
    chunks:
      latitude: 125
      longitude: 120
      time: 50
    combine: by_coords
    decode_times: true
    parallel: true
  meta:
    category: meteo
    notes: Extracted from Copernicus Climate Data Store
    paper_doi: 10.1002/qj.3803
    paper_ref: Hersbach et al. (2019)
    source_license: https://cds.climate.copernicus.eu/cdsapp/#!/terms/licence-to-use-copernicus-products
    source_url: https://doi.org/10.24381/cds.bd0915c6
    source_version: ERA5 hourly data on pressure levels
  path: meteo/era5/{variable}/era5_{variable}_{year}_hourly.nc
  rename:
    msl: press_msl
    ssrd: kin
    t2m: temp
    tisr: kout
    tp: precip
  unit_add:
    temp: -273.15
  unit_mult:
    kin: 0.000277778
    kout: 0.000277778
    precip: 1000
    press_msl: 0.01
```

Reply:
Nonetheless, fine with this extra functionality, just for your info.